core/modules/widgets/transclude.js

```diff
@@ -41,28 +41,45 @@ TranscludeWidget.prototype.execute = function() {
 	this.collectAttributes();
 	this.collectStringParameters();
 	this.collectSlotFillParameters();
-	// Get the target text and parse tree nodes that we are transcluding
-	var target = this.getTransclusionTarget(),
-		parseTreeNodes;
-	this.sourceText = target.text;
-	this.parserType = target.type;
-	this.parseAsInline = target.parseAsInline;
+	// Determine whether we're being used in inline or block mode
+	var parseAsInline = !this.parseTreeNode.isBlock;
+	if(this.transcludeMode === "inline") {
+		parseAsInline = true;
+	} else if(this.transcludeMode === "block") {
+		parseAsInline = false;
+	}
 	// Set 'thisTiddler'
 	this.setVariable("thisTiddler",this.transcludeTitle);
+	var parseTreeNodes, target;
 	// Process the transclusion according to the output type
 	switch(this.transcludeOutput || "text/html") {
 		case "text/html":
-			// Return the parse tree nodes
+			// Return the parse tree nodes of the target
+			target = this.parseTransclusionTarget(parseAsInline);
+			this.sourceText = target.text;
+			this.parserType = target.type;
+			this.parseAsInline = target.parseAsInline;
```

Comment:
It seems `this.parseAsInline` is only initialized if this case has been executed. Is that intentional? IMO undefined means "we do not know".  The variable `parseAsInline` is defined as true or false. Just a gut feeling.

Reply:
The code relies on this.parseAsInline defaulting to false